features/workspace/products/product/label-tags/toolbar/ToolbarMarkersButton.tsx

```
@@ -56,15 +61,22 @@ const ToolbarMarkersButton = ({
                 {isMarkerTypeGroup(markers) &&
                   markers.markerTypes.map((markerType) => (
-                    <Button
-                      variant="ghost"
-                      size="icon"
-                      key={markerType.name}
-                      title={markerType.name}
-                      onClick={() => handleMarkerSelection(markerType)}
-                    >
-                      <markerType.icon />
-                    </Button>
+                    <Tooltip>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)